packages/components/src/components/buttons/PinButton/PinButton.tsx

```diff
@@ -33,18 +40,22 @@ const Button = styled.button`
         content: ' ';
         position: absolute;
         border-radius: 100%;
```

Comment:
border-radius: ${borders.radii.full};

Reply:
Hmm, do we really want to change 100% to 100px?

Reply:
OK, let's keep it